DNN Platform/Library/Common/Globals.cs

```diff
@@ -3620,101 +3707,143 @@ public static string UserProfilePicRelativeUrl(bool includeCdv)
-        public static string FormatEmail(string Email)
+        public static string FormatEmail(string email)
         {
-            return HtmlUtils.FormatEmail(Email);
+            return HtmlUtils.FormatEmail(email);
         }
 
+        /// <summary>
+        /// Formats a domain name including link.
+        /// </summary>
+        /// <param name="website">The domain name to format.</param>
+        /// <returns>The formatted domain name.</returns>
         [Obsolete("This function has been replaced by DotNetNuke.Common.Utilities.HtmlUtils.FormatWebsite. Scheduled removal in v11.0.0.")]
-        public static string FormatWebsite(object Website)
+        public static string FormatWebsite(object website)
         {
-            return HtmlUtils.FormatWebsite(Website);
+            return HtmlUtils.FormatWebsite(website);
         }
 
+        /// <summary>
+        /// Xml encodes an html string.
+        /// </summary>
+        /// <param name="html">The html to encode.</param>
+        /// <returns>The encoded html for usage in xml.</returns>
         [Obsolete("This function has been replaced by DotNetNuke.Common.Utilities.XmlUtils.XMLEncode. Scheduled removal in v11.0.0.")]
-        public static string XMLEncode(string HTML)
+        public static string XMLEncode(string html)
         {
-            return XmlUtils.XMLEncode(HTML);
+            return XmlUtils.XMLEncode(html);
         }
 
+        /// <summary>
+        /// Gets the database connection string.
+        /// </summary>
+        /// <returns>The database connection string.</returns>
         [Obsolete("This function has been replaced by DotNetNuke.Common.Utilities.Config.GetConnectionString. Scheduled removal in v11.0.0.")]
         public static string GetDBConnectionString()
         {
             return Config.GetConnectionString();
         }
 
-        [Obsolete("This method has been deprecated. . Scheduled removal in v11.0.0.")]
-        public static ArrayList GetFileList(DirectoryInfo CurrentDirectory, [Optional, DefaultParameterValue("")] // ERROR: Optional parameters aren't supported in C#
-                                                                                string strExtensions, [Optional, DefaultParameterValue(true)] // ERROR: Optional parameters aren't supported in C#
-                                                                                bool NoneSpecified)
+        /// <summary>
+        /// Gets a file list.
+        /// </summary>
+        /// <param name="currentDirectory">The current directory.</param>
+        /// <param name="strExtensions">The extensions to filter for.</param>
+        /// <param name="noneSpecified">If true, adds 'None Specified' to the list.</param>
+        /// <returns>A list of file names.</returns>
+        [Obsolete("This method has been deprecated. Scheduled removal in v10.0.0.")]

```

Comment:
Why did this change to v10 instead of v11?

Reply:
Addressed in latest commit, thanks.